acceptance/lib/helper.rb

```diff
@@ -30,7 +30,8 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "ba39f2f2fd1b4def9a7f366d469acb1c5f6db824", :string)
+                         "1a2a63d86e8f7ee6cc354b988f49b36d007620f4",
```

Comment:
This looks like a Ruby Puppet repo SHA, not a Puppet-Agent package SHA.  Is this right?